policy/modules/system/systemd.te

```diff
@@ -1144,6 +1169,36 @@ optional_policy(`
 	dbus_watch_system_bus_runtime_named_sockets(systemd_resolved_t)
 ')
 
+#########################################
+#
+# Socket-proxyd local policy
+#
+kernel_read_system_state(systemd_socket_proxyd_t)
+
+auth_use_nsswitch(systemd_socket_proxyd_t)
+sysnet_dns_name_resolve(systemd_socket_proxyd_t)
+
+# resolver
+allow systemd_socket_proxyd_t self:unix_dgram_socket { create getopt setopt sendto read write };
+
+# listener
+corenet_port(systemd_socket_proxyd_port_t)
+allow systemd_socket_proxyd_t self:tcp_socket accept;
```

Comment:
Please group all of the raw allow rules at the top of the section -- before the `kernel_read_system_state()`